djangocms_url_manager/forms.py

```diff
@@ -185,22 +199,33 @@ def clean_anchor(self):
 
     def save(self, **kwargs):
         url_type = self.cleaned_data.get("url_type")
+        url = super().save(commit=False)
+        commit = kwargs.get("commit", True)
         is_basic_type = url_type in dict(BASIC_TYPE_CHOICES).keys()
         if is_basic_type:
             # Set content object to none to prevent GFK url always being returned by getter.
             self.instance.content_object = None
         else:
-            self.instance.content_object = self.cleaned_data["content_object"]
-        return super().save(**kwargs)
+            self.instance.content_object = self.cleaned_data.get("content_object")
+
+        # Check whether the form used has the url_grouper attribute, as overrides do not.
+        if hasattr(url, "url_grouper"):
+            if not getattr(url, "url_grouper"):
+                url.url_grouper = UrlGrouper.objects.create()
+        if commit:
+            url.save()
+        return url
 
 
 class UrlOverrideForm(UrlForm):
     class Meta:
         model = UrlOverride
-        fields = ("url",) + UrlForm.Meta.fields
+        base_fields = UrlForm.Meta.fields
```

Comment:
Reverse the changes to this:

```suggestion
        fields = ("url",) + UrlForm.Meta.fields
```